.github/workflows/Docs.yml

```
@@ -0,0 +1,42 @@
+name: Docs
+
+on:
+  push:
+    branches: [master]
+    tags: '*'
+  pull_request:
+
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+
+jobs:
+  docs:
+    name: Documentation
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      statuses: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: '1'
+      - name: Configure doc environment
+        run: |
+          julia --project=docs/ -e '
+            using Pkg
+            Pkg.develop(PackageSpec(path=pwd()))
+            Pkg.instantiate()'
+      - uses: julia-actions/julia-buildpkg@v1
```

This should just be
```suggestion
      - uses: julia-actions/julia-buildpkg@v1
```
The `Pkg.develop` etc. is already part of the `julia-docdeploy` action: https://github.com/julia-actions/julia-docdeploy/blob/580a86b97cef20258d96b53292326e6258dd6859/action.yml#L28-L56